src/model/Post.ts

```diff
@@ -2,20 +2,11 @@ import {Category} from "./Category";
 
 export class Post {
 
-    id: number;
-    title: string;
-    text: string;
-    createDate: Date = new Date();
-    categories: Category[];
-
-    constructor(id: number,
-                title: string,
-                text: string,
-                categories: Category[]) {
-        this.id = id;
-        this.title = title;
-        this.text = text;
-        this.categories = categories;
-    }
+    constructor(
+        public id: number,
+        public title: string,
+        public text: string,
+        public categories: Category[]
+    ) { }
```

Comment:
nope lets left it verbose. verbose is good because its demo and should be as simple as possible for users